pom.xml

```diff
@@ -1425,11 +1441,16 @@
               <release>${maven.compiler.release}</release>
               <showWarnings>true</showWarnings>
               <compilerArgs>
+                <arg>-J--add-exports=jdk.compiler/com.sun.tools.javac.api=ALL-UNNAMED</arg>
```

Comment:
Now matches the `errorprone` installation guide.
https://errorprone.info/docs/installation